python/nav/metrics/thresholds.py

```diff
@@ -230,18 +234,13 @@ class InvalidExpressionError(Exception):
     pass
 
 
-def interval_to_graphite(delta):
+def interval_to_graphite_start(delta):
     """Converts a timedelta to a usable Graphite time specification.
 
     :type delta: datetime.timedelta
 
     """
-    secs = delta.total_seconds()
-    if secs % YEAR == 0:
-        return "{0}year".format(int(secs / YEAR))
-    elif secs % DAY == 0:
-        return "{0}day".format(int(secs / DAY))
-    elif secs % MINUTE == 0:
-        return "{0}min".format(int(secs / MINUTE))
-    else:
-        return "{0}s".format(int(secs))
+    now = datetime.now(tz=ZoneInfo(settings.TIME_ZONE))
+    start = now - delta
+    graphite_start = start.strftime(GRAPHITE_TIME_FORMAT)
+    return graphite_start
```

Comment:
I'd like to know why the previous version was so clunky, maybe graphite got a more sensible format and it is technical debt?